web-ui/backend.mli

```diff
@@ -1,7 +1,13 @@
+(** Handles connections to ocaml-ci-service via Capnp *)
+
 open Capnp_rpc_lwt
 
 type t
 
 val make : Ocaml_ci_api.Raw.Client.CI.t Sturdy_ref.t -> t
 
 val ci : t -> Ocaml_ci_api.Client.CI.t Lwt.t
+
+module Make : sig
+  val ci : Uri.t -> Ocaml_ci_api.Client.CI.t Lwt.t
+end
```

Comment:
Why do we need to expose `ci` and `Make.ci`? What is the difference between them apart from the Uri.t?

Reply:
I guess the implementation enables this kind of thing and I've been playing with the notion that this is idiomatic OCaml 🤷🏾‍♂️ 

```
    Backend.Make.ci backend_cap >>= fun ci ->

```